src/components/SelectionList/BaseListItem.tsx

```diff
@@ -84,20 +82,38 @@ function BaseListItem({
                             </View>
                         </View>
                     )}
-                    <ListItem
-                        item={item}
-                        textStyles={[
-                            styles.optionDisplayName,
-                            isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText,
-                            isUserItem || item.isSelected || item.alternateText ? styles.sidebarLinkTextBold : null,
-                            styles.pre,
-                            item.alternateText ? styles.mb1 : null,
-                        ]}
-                        alternateTextStyles={[styles.optionAlternateText, styles.textLabelSupporting, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, styles.pre]}
-                        isDisabled={isDisabled}
-                        onSelectRow={onSelectRow}
-                        showTooltip={showTooltip}
-                    />
+
+                    {item.rightElement === undefined ? (
```

Comment:
Is this necessary? It looked, cleaner before

Reply:
Unfortunately there is no way to make it shorter e.g. as it was before because of type difference between those components